satellite/orders/service.go

```diff
@@ -596,15 +604,16 @@ func (service *Service) CreatePutRepairOrderLimits(ctx context.Context, repairer
 		}
 
 		orderLimit, err := signing.SignOrderLimit(ctx, service.satellite, &pb.OrderLimit2{
```

Comment:
It looks like orderlimit content has been changed but the signing and verification not.

Reply:
what else needs to change?